lib/f_service/result/success.rb

```diff
@@ -9,20 +9,20 @@ module Result
     #
     # @!attribute [r] value
     #   @return [Object] the provided value for the result
-    # @!attribute [r] type
-    #   @return [Object] the provided type for the result. Defaults to nil.
+    # @!attribute [r] types
+    #   @return [Object] the provided types for the result. Defaults to nil.
     # @api public
     class Success < Result::Base
-      attr_reader :value, :type
+      attr_reader :value, :types
```

Comment:
@bvicenzo I've tested the new version on Kippu and the only incompatibility was the lack of a `type` method. With the change to `types`, situations where the type of the result is checked don't work anymore, eg.:

```rb
result = Service.(:foo)

result.type == :foo
```

Reply:
@matheusbsilva Man, pode atualizar e verificar novamente, por favor?

Reply:
@matheusbsilva I've checked and fixed, it here. It is working, now.